### PR TITLE
add config for vault lookup paths

### PR DESCRIPTION
**Pull request by vito**
for https://github.com/concourse/concourse/pull/5013

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -467,6 +467,10 @@ concourse:
       ##
       sharedPath:
 
+      ## Path templates to use for credential lookup.
+      ##
+      lookupTemplates: ["/{{.Team}}/{{.Pipeline}}/{{.Secret}}", "/{{.Team}}/{{.Secret}}"]
```
Comment (vito) [inline values.yaml:472]: I'm iffy on having a default here but it looks like that's what we do elsewhere. 🤔

Comment (taylorsilva) [reply values.yaml:472]: What happens if it's blank? does the binary have a default embedded in it?

Comment (vito) [reply values.yaml:472]: Yep. Should I just remove this?

Comment (vito) [reply values.yaml:472]: (Yes.)

**Pull request merged**